### PR TITLE
Media collection 174 mods

### PR DESCRIPTION
**Pull request by mattheu**
Updates to gallery field

* Change markup to remove text and 'remove' links for single images.
* Enable sorting of the gallery items.

Note that this is a massive diff because I merged in master - I have a separate PR for this - https://github.com/alleyinteractive/wordpress-fieldmanager/pull/293

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@mattheu hmm. maybe we just need to start a new branch?

**Comment by mattheu** [comment]
OK - this looks better

File: js/media/fieldmanager-media.js [hunk 1]
```diff
@@ -132,15 +176,30 @@ $( document ).on( 'click', '.fm-media-button', function( event ) {
 			props.link = 'custom';
 			props.linkUrl = '#';
 			props.caption = '';
+
+			preview += '<div class="media-item" data-id="' + attachment.id + '">';
```
Comment (danielbachhuber) [inline js/media/fieldmanager-media.js]: Need to build this with jQuery, lest VIP get on our case about it.

**Comment by danielbachhuber** [comment]
I'll go ahead and merge this, and itemize the remaining issues on the first PR.

**Pull request merged**